### ISSUE TITLE
Update the Wiki

### ISSUE THREAD

**Issue by BlackFisch** [AsYetUntitled/Framework#645]
I noticed that the wiki is a bit out of date or unclear in some parts and should be updated. Possible changes include:

- Removing instructions for older versions (extDB2 legacy stuff)
- Providing a Script for SteamCMD (for updating arma and such)
- Syntax-Highlighting for Code-Blocks
- Better styling
- Better Tanoa Life explanations & integration in server.cfg
- Installation Instructions for Headless Client
- Creating firewall rules (Thanks, Windows)

Because Github doesn't allow Wiki-Forking (why tho?) I created a gist implementing the above mentioned changes, which could be merged into this wiki: https://gist.github.com/BlackFisch/98f4f2d48a15e164af785a1fc4692fd5

If you have any suggestions, let me know :)

**Comment by BlackFisch** [comment]
@BoGuu @Jawshy :)

**Comment by BoGuu** [comment]
Diff for reference:

https://gist.github.com/BoGuu/29fe5638459799857f6b5660bb0ff27d

**Comment by BlackFisch** [comment]
Updated the Gist to check for HC precense before starting it
(also added updated diff to the gist)
